fw/http.c

```diff
@@ -6711,6 +6713,7 @@ tfw_http_resp_process(TfwConn *conn, TfwStream *stream, struct sk_buff *skb)
 	 * protocol data.
 	 */
 	if (skb && !websocket) {
+		*splitted = NULL;
```

Comment:
In this and the above cases we explicitly set `splitted` to `NULL`, but there are return paths in the function, which do not touch `splitted` at all. I'd propose to leave only `*splitted = skb`.

BTW split is irregular verb with the same form, split split split. Could you please also fix this in `tfw_http_req_process()`?

Reply:
We should zeroed `splitted` because if it will not be zeroed we will free it in `tfw_connection_recv` later